include/aikido/perception/ObjectDatabase.hpp

```diff
@@ -41,10 +47,17 @@ class ObjectDatabase
 
   virtual ~ObjectDatabase() = default;
 
+  /// Get the object name, resource, and offset from database by objectKey
+  /// \param[in]  objectKey: The key(string) of an object in ObjectDatabase
```

Comment:
I believe we don't want colons in Doxygen comments. Also, minor typo: `key(string)` -> `key (string)`.